ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/port-channel-interfaces.j2

```diff
@@ -20,41 +20,48 @@ port_channel_interfaces:
   Port-Channel{{ channel_group_id }}:
 {%                         set peer = connected_endpoint %}
     description: "{% include switch.interface_descriptions.connected_endpoints_port_channel_interfaces %}"
-    type: switched
+{%                         if adapter_settings.port_channel.subinterfaces is arista.avd.defined %}
```

Comment:
Considering the worst case of user definition, I would check if the list is not empty.

For example: ```adapter_settings.port_channel.subinterfaces: []```

Reply:
I don't think we need that level of protection. Worst case is that it outputs a routed interface with no subinterfaces. And only because the user set the key `subinterfaces`.

Reply:
Yes.